gradle.properties

```diff
@@ -0,0 +1,2 @@
+android.enableJetifier=true
+android.useAndroidX=true
```

Comment:
It's typically not a good idea to have `gradle.properties` checked into git since conventionally it's used to store sensitive information. In WPAndroid and WCAndroid, we have this file in `.gitignore` [#](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/.gitignore#L52), [#](https://github.com/woocommerce/woocommerce-android/blob/develop/.gitignore#L24) and instead utilize a `gradle.properties-example` file. For the CI, we use a step to copy the example file, or decrypt one with actual values depending on the task.

We unfortunately have some libraries that does this exact same thing, but we should be changing those as well. It's not harmful if everyone is careful not to push anything sensitive, but there is no reason to risk it.

_P.S: I think it was a poor decision to require these properties in the `gradle.properties` file 🤷‍♂️_

Reply:
Thanks, TIL 
> conventionally it's used to store sensitive information

I've never considered this file as a storage for any sensitive information, but if anyone does, I'm ok with having this in `.gitignore` and use strategy from `WPAndroid` or `WCAndroid` you mention.

Reply:
You may want to check out the `gradle.properties` file in your WPAndroid and WCAndroid folders :) You are probably using our configuration tool to fetch/update the secrets, so you may not have added anything sensitive yet, but they are certainly there.

Whether that's the perfect place for them is questionable to me as well, but it's the convention and currently what we use. On one hand, it makes it a lot easier to access the properties since Gradle will automatically load them from the `gradle.properties` file, but on the other, it might be better to have a separate file for those to avoid this type of confusion. 🤷‍♂️

Reply:
Sure, that's good idea to follow the same convention as in other projects. I've addressed this in #62